audit-opentelemetry/plugindefinition.yaml

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: greenhouse.sap/v1alpha1
+kind: PluginDefinition
+metadata:
+    name: opentelemetry-audit
+spec:
+    version: 0.6.1
+    displayName: OpenTelemetry for Audit Logs
+    description: Audit Logs relevant Observability framework for instrumenting, generating, collecting, and exporting telemetry data such as traces, metrics, and logs.
+    icon: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/opentelemetry/logo.png
```

Comment:
Maybe we can change the icon so the difference between the two plugins becomes recognizable?